ethereum/extethclient/eth_wallet_client.go

```diff
@@ -20,6 +21,12 @@ import (
 	"github.com/athanorlabs/atomic-swap/ethereum/block"
 )
 
+const (
+	mainnetChainID = 1
+	goerliChainID  = 5
+	ganacheChainID = 1337
+)
```

Comment:
these are also defined in https://github.com/AthanorLabs/atomic-swap/pull/262 - whichever gets merged second should make sure there's no duplicates